unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2804,7 +2804,7 @@ float allowedError
             SimObjPhysics soInHand = ItemInHand.GetComponent<SimObjPhysics>();
 
             Vector3 handObjPosRelAgent =
-                Quaternion.Euler(0, rotation - transform.rotation.y, 0) *
+                Quaternion.Euler(0, rotation - transform.eulerAngles.y, 0) *
```

Comment:
This is unrelated to this PR, but a bug that I discovered that uses `transform.rotation.y` instead of `transform.eulerAngles.y`, which are completely different and results in a silent bug.

Reply:
Good catch!